frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -662,6 +662,7 @@ local VirtualKeyboard = FocusManager:new{
 
     lang_to_keyboard_layout = {
         ar_AA = "ar_AA_keyboard",
+        fa = "fa_keyboard",
```

Comment:
Could you move this down to its alphabetic location for ease of maintenance?

Reply:
hi 
i fix that :)